di_website/publications/models.py

```diff
@@ -368,6 +368,12 @@ class Meta:
         FieldPanel('colour'),
         hero_panels(),
         StreamFieldPanel('authors'),
+        MultiFieldPanel([
+            FieldPanel('call_to_action_title'),
+            FieldPanel('call_to_action_body'),
+            FieldPanel('call_to_action_button_text'),
+            FieldPanel('call_to_action_button_url'),
+        ], heading='Call to Action Section'),
```

Comment:
Create a `callToActionPanel` function to go with the mixin

Reply:
Added cta panel for publications and blogs